### PR TITLE
[Sky]Gods Additional Effects

### PR DESCRIPTION
**Pull request by nesstea**

### CODE REVIEW

File: scripts/zones/RuAun_Gardens/mobs/Suzaku.lua [hunk 1]
```diff
@@ -36,5 +46,30 @@ function onMonsterMagicPrepare(mob, target)
...
+    local dmg = statDiff + levelDiff + math.random(0, 15); -- INT modifier + difference in level + variance
+    local params = {};
+    params.bonusmab = 0;
+    params.includemab = false;
+    
+    dmg = addBonusesAbility(mob, ELE_FIRE, target, dmg, params);
+    dmg = dmg * applyResistanceAddEffect(mob, target, ELE_FIRE, 0);
+    dmg = adjustForTarget(target, dmg, ELE_FIRE);
+    
+    if (dmg < 0) then
```
Comment (TeoTwawki) [inline scripts/zones/RuAun_Gardens/mobs/Suzaku.lua]: If less than zero, change the message id (with the right gear, you can in fact get healed by the "damage").

**Comment by TeoTwawki** [comment]
To view diff without the whitespace changes: https://github.com/DarkstarProject/darkstar/pull/1328/files?w=1

File: scripts/globals/items/antarctic_windy.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
+-----------------------------------------
+-- ID: 18164
+-- Item: Antarctic Wind
```
Comment (TeoTwawki) [inline scripts/globals/items/antarctic_windy.lua]: @nesstea  *D'oh!* I typo'd the file name. "Antarctic_windy.lua"  ... /FAIL

Comment (nesstea) [inline scripts/globals/items/antarctic_windy.lua]: LOL ok 1 sec

**Comment by teschnei** [comment]
you don't need to check dmg < 0 and change the messageid if it is, the core automatically does that for you,  so remove those and it'll be good

(you changed it to the wrong messageid anyways)

**Comment by TeoTwawki** [comment]
@teschnei unless something was recently changed, it won't auto change that because Action->addEffectMessage is set by that return. Which is why all pre-existing scripts did that. As for the ID, guessing when I assisted I went and used message ID for dmg/heal on mob target instead of the message for on player target.

**Comment by teschnei** [comment]
i don't want to make every script writer remember to do that manually, so if there is a problem, then it needs to be fixed in the core and not in 100 different scripts

**Comment by TeoTwawki** [comment]
@teschnei not it :)

**Comment by nesstea** [comment]
Doing another pull req because I derped. Changes in next.

**Pull request closed without merging**